### ISSUE TITLE
Admin\Operator - Internal User can view "Operator Information"

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1700]
Partially blocked by #1699 

#### Description:
As an internal user, I can see the information for an operator

#### Acceptance Criteria:
Given I am on [operator details page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=14652-3185&t=EdbpdHgRnvGxWEpV-4) handled in ticket #1699 
When I click the Operator Information tile
Then I am brought to the operator information page 
* same as external user view, but in view-only mode, with no edit button
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18028-8029&t=EdbpdHgRnvGxWEpV-4)

#### Development Checklist:

- [x] Create page in folder  `bciers/apps/administration/app/idir/cas_admin/operators/[operatorId]/page.tsx` to ` return <OperatorPage />;`  
- [x] Also(?), Create page in folder `bciers/apps/administration/app/idir/cas_analyst/operators/[operatorId]/page.tsx` to ` return <OperatorPage />;`  
- [x] implement logic in `OperatorForm` for `SingleStepTaskListForm` property  `allowEdit={role === FrontEndRoles.INDUSTRY_USER_ADMIN}`
- [x] Ensure `bc_obps/common/fixtures/dashboard/?` fixture created in #1699 for operator details has link to `/administration/operators/stub-id` 
- [x] Implement function to replace the static href, `/administration/operators/stub-id`, with dynamic href of the Operator ID obtained from the app router parameters (eg. https://github.com/bcgov/cas-registration/pull/1943/files#diff-75b8bf405e1f0dbab3d2218568846b7d4de7a5c78e86105644124064be3d025c)
- [x] modify `OperatorForm` vitests to reflect internal view-only mode, with no edit button
- [x] Delete page `bciers/apps/administration/app/idir/cas_admin/operators/user-operator/[id]/[formSection]/page.tsx`- [ ] Delete page `bciers/apps/administration/app/idir/cas_analyst/operators/user-operator/[id]/[formSection]/page.tsx`
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by andrea-williams** [comment]
@zoeyli-46 could you please update the Figma links in this ticket? They seem to be linked to COAM wireframes

**Comment by zoeyli-46** [comment]
its updated
